.gitignore

```diff
@@ -11,3 +11,5 @@ project/plugins/project/
 
 # Scala-IDE specific
 .scala_dependencies
+
+.idea
```

Comment:
we usually don't have editor-specific stuff in the gitignore, the above actually prompted me to log #75

Reply:
Checked!